UKHomeOffice/cop-product-page
# COP-9212: Override immer version

### Description
Added a resolution to `package.json` for `immer` version `9.0.6`, addressing a security issue that was failing the npm audit.

This also required a preinstall step to force the resolutions (that is out-of-the-box functionality in yarn but not npm).

### To test
As described, this addresses a security issue with versions of `immer` earlier than `9.0.6`. This should now build and deploy correctly without the npm audit failing. It will need some basic regression tests on top of ensuring that the broken functionality (with the earlier version of `immer`) is now working again.
